VERSION

```diff
@@ -1 +1 @@
-0.4.0-dev
```

Comment:
Look at [build/bump-release](build/bump-release). I'm not a huge fan of this way of doing it either, but that's the current setup.